SRGBmods_Pico_LED_Controller/README.md

```diff
@@ -0,0 +1,28 @@
+![](https://srgbmods.net/picoledcontroller/img/PicoLEDController.png)
+### SRGBmods Pico LED Controller for SignalRGB
+
+This is an open-source DIY (do it yourself) project to create a small, affordable - yet still powerful - addressable RGB controller that can be managed by **SignalRGB** with sending packets via USB HID to the Pico.
+
+In the **/src/** folder you will find:
+- the **SRGBmods_Pico_LED_Controller.ino** sketch to compile with Arduino IDE
```

Comment:
The examples in this `pico-examples` repo are all designed to be used in conjunction with https://github.com/raspberrypi/pico-sdk

I believe any Arduino-based example-projects will need to be added to whatever the appropriate Arduino-projects website would be.